src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -270,18 +267,14 @@ class BtreeLBAManager : public LBAManager {
   alloc_extent_ret alloc_extent(
     Transaction &t,
     laddr_t hint,
-    extent_len_t len,
-    paddr_t addr,
-    uint32_t checksum,
     LogicalCachedExtent &ext) final
   {
     return _alloc_extent(
       t,
       hint,
-      len,
-      addr,
-      P_ADDR_NULL,
-      checksum,
+      ext.get_length(),
+      ext.get_paddr(),
+      ext.get_last_committed_crc(),
```

Comment:
Um, this assert can fail if a fully-loaded extent is remapped.

Or maybe we should delay the crc calculation of remapped extents until the transaction commit. What do you think?

Reply:
> Or maybe we should delay the crc calculation of remapped extents until the transaction commit. What do you think?

Yeah, so that all the fresh extents including the remapped ones are consistent.